Week3/1. Strings.js

```diff
@@ -0,0 +1,16 @@
+/***********************************  Strings ***************************************************/
+
+const myString = "hello,this,is,a,difficult,to,read,sentence";
+
+//  Add the string to your file and log it.
+console.log(myString); // hello,this,is,a,difficult,to,read,sentence
+
+
+//  Log the length of myString.
+console.log(myString.length); // 42
+
+//  Replace commas with spaces ????
+let myString2 = myString.replace(/,/g , ' ');
+console.log(myString2);  // hello this is a difficult to read sentence
```

Comment:
Nice! Do you think its better to use `let` here, or `const`?

Reply:
Hi Marciano, first of all, I am deeply sorry for my late response.  I tried to make all implements according to your feedback.
I think, using let is better here since it allows to change in string in case of re-assign needed.

Reply:
Hi Marciano, first of all, I am deeply sorry for my late response.  I tried to make all implements according to your feedback.
I think, using let is better here since it allows to change in string in case of re-assign needed.